src/pybind/mgr/cephadm/module.py

```diff
@@ -2979,7 +2979,7 @@ def list_specs(self) -> orchestrator.Completion:
         specs = list()
         for service_name, spec in self.spec_store.specs.items():
             specs.append('---')
-            specs.append(yaml.dump(spec))
+            specs.append(spec.to_yaml())
```

Comment:
actually, instead of introducing a new `to_yaml` to every data structure, why not simply

```suggestion
            specs.append(yaml.dump(spec.to_json()))
```
? This way we can keep yaml contained in the orch cli